conf/compose/docker-compose.memcached.yml

```diff
@@ -5,3 +5,5 @@ services:
     command: ["-m", "1024m"]
```

Comment:
This differs from the Core-Portal - does that matter?
https://github.com/TACC/Core-Portal/pull/828/files#diff-67ee10ce7fc260521e9912a611ab1003a513c09ec83ddf5c5ed2c74ab198d82aR36

Reply:
This is a good question, this option sets the memory usage for memcache, is it necessary for us to do that? Should we do it for both or not worry about it? Open to thoughts here.

Reply:
I've done a little reading, it seems that the default memory limit of memcache is 64kb, so it might be useful to set a higher limit. I can update the Core-Portal docker settings file to match with this one if that sounds good to everyone. Also, should we look into more detailed monitoring for it? I can create a Jira ticket for this.

Reply:
Nice find. 64kb might be ok, as I'm not actually sure we're even using it in the code. If it relies on us using the`django.core.cache`, we don't have that in use, so maybe we're ok for now?

Reply:
That sounds fine to me, I don't really have strong feelings either way and we can always change it later if necessary. I'll make sure both are updated to just use the default.